### PR TITLE
Add Longest Function

### PR DESCRIPTION
**Pull request by ChrisOwen101**
This PR adds a function that finds the longest string

### CODE REVIEW

File: longest.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+def find_longest(arr):
```
Comment (qrafiq1) [inline longest.py:1]: function docstring

**Comment by Kurt812** [comment]
Could you have added a description?

**Review by GabrielNsiah — APPROVED** [review]
I'm sure it works

**Comment by keoghrmj** [comment]
Could this be done in one line?

File: longest.py (hunk 1 above)
Comment (RidwanHamid501) [inline longest.py:1]: docstrings?

**Review by Kurt812 — APPROVED** [review]
Looks good

File: longest.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    longest = arr[1]
```
Comment (S1mpySloth) [inline longest.py:2]: Could this have a docstring?

**Review by anitamegarry — COMMENTED** [review]
Code does not work

**Review by Jakub-Poskrop — COMMENTED** [review]
Shouldn't the return not be in the for loop?

**Pull request merged**